internal/wrappers/results-modifier.go

```diff
@@ -48,19 +50,46 @@ func (s *ScanResult) UnmarshalJSON(data []byte) error {
 func (s *ScanResultNode) UnmarshalJSON(data []byte) error {
 	type Alias ScanResultNode
 	aux := &struct {
-		Column int `json:"column"`
+		Column     int    `json:"column"`
+		Line       int    `json:"line"`
+		Length     int    `json:"length"`
+		MethodLine int    `json:"methodLine"`
+		FileName   string `json:"fileName,omitempty"`
 		*Alias
 	}{
 		Alias: (*Alias)(s),
 	}
 	if err := json.Unmarshal(data, &aux); err != nil {
 		return err
 	}
-
 	s.Column = 0
+	s.Line = 0
+	s.Length = 0
+	s.MethodLine = 0
+	message := "Found negative %v with value %v in file %v"
 	if aux.Column >= 0 {
 		s.Column = uint(aux.Column)
+	} else {
+		message = fmt.Sprintf(message, "column", aux.Column, aux.FileName)
+		logger.PrintIfVerbose(message)
+	}
+	if aux.Line >= 0 {
+		s.Line = uint(aux.Line)
+	} else {
+		message = fmt.Sprintf(message, "line", aux.Line, aux.FileName)
```

Comment:
What if column was also negative and message is already reassigned? We should set the format as a constant and do:
```
logger.PrintIfVerbose(fmt.Sprintf(...))
```

We can also define a function that receives an int and contains the if/else statements, returning the value casted to uint or logging and returning 0. Like this we can cut the if/else repetition and do:
```
s.Column = function(aux.Column)
s.Line = ...  
...  
```

Reply:
Good point, but the message should still be printed in all cases I think. But will check and remove the if repetition with an aux function. Thanks !

Reply:
But in the previous "else" we are doing `message = fmt.Sprintf(message, "line", aux.Line, aux.FileName)` so in the next if/else  the message will not have the %v format placeholders right?

Reply:
Yes, I was checking and failed. I will change that